src/index.js

```diff
@@ -4,18 +4,24 @@ const { createElement } = require('react');
 const { renderToString } = require('react-dom/server');
 const { ServerStyleSheet } = require('styled-components');
 
-const page = (title, styles, body) => `<!DOCTYPE html>
+const h = createElement;
+
+const page = (title, styles, body, bundle, props) => `<!DOCTYPE html>
 <html>
   <head>
     <title>${title}</title>
     ${styles}
   </head>
   <body style="margin: 0;">
-    ${body}
+    <div id="root">
+      ${body}
+    </div>
+    ${props ? `<script>window.appProps = ${JSON.stringify(props).replace(/</g, '\\u003c')};</script>`: ''}
+    ${bundle ? `<script type="text/javascript" src="${bundle}" />`: ''}
```

Comment:
I think something a bit more descriptive and less specific to our implementation might be better here: ~bundle~ => `clientCode`

Reply:
I think 'bundle' is a fairly standard term for it. - https://medium.com/@phoebe.greig/headache-free-isomorphic-app-tutorial-react-js-react-router-node-js-ssr-with-state-and-es6-797a8d8e493a

Reply:
Or do you just mean for the `page()` function?

Reply:
I guess generally. I mean this now constitutes part of the API of `lev-react-renderer` (which presumably doesn't enforce or expect bundling), so a less opinionated name would be good... furthermore, it might be nice if this setting could also be an array  of scripts, in case people have multiple sources (e.g. various files served by CDNs in addition to the consumers' app code).

Reply:
Let's have a separate argument for this.

Reply:
Updated.